.github/workflows/ci-arm64.yml

```diff
@@ -0,0 +1,97 @@
+          install: |
+            set -x
+            apt-get update -q -y
+            apt-get install -q -y make gcc g++ clang libgd-dev libgeoip-dev libxslt1-dev libpcre++0v5 libpcre++-dev liblua5.1-0-dev lua5.1 libperl-dev cpanminus libssl-dev
+          run: |
+            set -x
+            cd /tengine
+            echo "Build luajit2"
+            cd luajit2
+            make -j2
+            make install
+            cd ..
+            echo "Build tengine"
+            export CC=${{ matrix.compiler.CC }}
+            export CXX=${{ matrix.compiler.CXX }}
+            export LUAJIT_LIB=/usr/local/lib
+            export LUAJIT_INC=/usr/local/include/luajit-2.1
+            ./configure \
+              --with-ld-opt="-Wl,-rpath,/usr/local/lib" \
+              --with-ipv6 \
+              --with-http_ssl_module \
+              --with-http_v2_module \
+              --with-http_addition_module \
+              --with-stream \
+              --with-stream_ssl_module \
+              --with-stream_realip_module \
+              --with-stream_geoip_module \
+              --with-stream_ssl_preread_module \
+              --with-stream_sni \
+              --add-module=./modules/ngx_backtrace_module \
+              --add-module=./modules/ngx_debug_pool \
+              --add-module=./modules/ngx_debug_timer \
+              --add-module=./modules/ngx_http_concat_module \
+              --add-module=./modules/ngx_http_footer_filter_module \
+              --add-module=./modules/ngx_http_lua_module \
+              --add-module=./modules/ngx_http_proxy_connect_module \
+              --add-module=./modules/ngx_http_reqstat_module \
+              --add-module=./modules/ngx_http_slice_module \
+              --add-module=./modules/ngx_http_sysguard_module \
+              --add-module=./modules/ngx_http_trim_filter_module \
+              --add-module=./modules/ngx_http_upstream_check_module \
+              --add-module=./modules/ngx_http_upstream_consistent_hash_module \
+              --add-module=./modules/ngx_http_upstream_dynamic_module \
+              --add-module=./modules/ngx_http_upstream_dyups_module \
+              --add-module=./modules/ngx_http_upstream_keepalive_module \
+              --add-module=./modules/ngx_http_upstream_session_sticky_module \
+              --add-module=./modules/ngx_http_upstream_vnswrr_module \
+              --add-module=./modules/ngx_http_user_agent_module \
+              --add-module=./modules/ngx_multi_upstream_module \
+              --add-module=./modules/ngx_slab_stat \
+              --without-http_upstream_keepalive_module
+            make -j2
+            make install
+            echo "Test..."
+            cd tests/nginx-tests
+            export TEST_NGINX_BINARY=/usr/local/nginx/sbin/nginx
+            cpanm --notest Net::DNS::Nameserver > build.log 2>&1 || (cat build.log && exit 1)
+            prove -v -Inginx-tests/lib $(ls -d tengine-tests/* | grep -v resolver_file)
```

Comment:
dont need to run cases here, too long time to wait, jst to build it for CI is ok

Reply:
@chobits I have removed the tests!

Github Actions does not provide (yet) cloud runners on ARM64, so we need to use emulation via QEMU, and this is few times slower indeed!